docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3414 @@
+* Flamegraph legend {kibana-pull}147910[#147910]
+* Adds api key to APM package policies {kibana-pull}147650[#147650]
+* Only renders waterfall items up until 3 levels {kibana-pull}147569[#147569]
+* Improve span links navigation {kibana-pull}147426[#147426]
+* Update default refresh interval to 60 seconds {kibana-pull}146791[#146791]
+* Adds pagination to source map API {kibana-pull}145959[#145959]
+* Ability to offset point labels on maps {kibana-pull}145773[#145773]
+
+Dashboard::
+* Unified panel options pane {kibana-pull}148301[#148301]
+* Anchor time slider to start {kibana-pull}148028[#148028]
+* Show document count beside options list suggestions {kibana-pull}146241[#146241]
+
+Discover::
+* Show "Copy value" button for any grid cell {kibana-pull}149525[#149525]
+* Integrate unified field list filters into Discover {kibana-pull}148547[#148547]
+* Persist field list sections state in local storage {kibana-pull}148373[#148373]
+* Enable adhoc data views creation from no data views state {kibana-pull}147850[#147850]
+* Adds a way to quickly expand time range from "No results" screen {kibana-pull}147195[#147195]
+* Optimize checking for multifields during grid rendering {kibana-pull}145698[#145698]
+* Integrate unified field list sections into Discover {kibana-pull}144412[#144412]
+* Update unified histogram to use Lens {kibana-pull}143117[#143117]
+
+Elastic Security::
+For the Elastic Security 8.7.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.7.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Fixes discard changes link taking user to "page not found" {kibana-pull}150174[#150174]
+* Adds filebeat_input index to agent policy default {kibana-pull}149974[#149974]
+* Bugfix: Apply namespace from agent policy if there is one when adding integration {kibana-pull}149949[#149949]
+* Agent List: Inform users when agents have become inactive since last page view {kibana-pull}149226[#149226]
+* Experimental toggles for doc-value-only {kibana-pull}149131[#149131]
+* Create index templates and ingest pipeline at package policy creation time for input packages {kibana-pull}148772[#148772]
+* Do not allow namespace or dataset to be edited for input only package policies {kibana-pull}148422[#148422]
+* Adds deprecation warning to unenrollment timeout agent policy setting {kibana-pull}147963[#147963]
+* Adds active filter count to agent status filter {kibana-pull}147821[#147821]
+
+Kibana UI::
+The list view in Dashboard, Visualize Library, Maps, and Graph now stores the latest state of the table in the URL {kibana-pull}145517[#145517]
+
+Lens & Visualizations::
+* Allows case sensitive option on multiselection filters input {kibana-pull}149570[#149570]
```

Comment:
This is also a unified search improvement which will be available wherever the component lives